### ISSUE TITLE
Improve the behavior of the tabulated_simulation_settings function

### ISSUE THREAD

**Issue by kai-zenk** [tudat-team/tudat#158]
When setting up an estimation environment using the tabulated_simulation_settings (https://py.api.tudat.space/en/latest/observation.html#tudatpy.numerical_simulation.estimation_setup.observation.tabulated_simulation_settings) function the default value of its reference_link_end_type argument (default = LinkEndType.receiver) sometimes (read: especially when using the cartesian_position observable)happens to clash with the value expected based on the observable_type argument. Moreover, it is difficult to track the error back to this function, since the compiler only complains in the line where one is actually these settings. It would be much more intuitive if the function was able to automatically choose the right value for the reference_link_end_type since this information could be inferred from the observable_type.

**Comment by DominicDirkx** [comment]
This is (kind of) already implemented, but not used in the Python exposure. If you use  unidentified_link_end as reference reference_link_end_type, it will go to a function retrieving the default reference link end type. To properly use this in Tudatpy, we should:

- Modify the default input to the  reference_link_end_type
- Expose the default_link_end enum entry (which is just an alias for  unidentified_link_end), but more suitable in this case
- Expose and document the function that retrieves the default link end, so that people know what's being used 

Would you have time to take care of the above points? I can show you the main steps during our next meeting

**Comment by kgzenk** [comment]
Yes - I can take care of these points (provided my tudat-bundle somehow magically decides to finally work...)

**Comment by DominicDirkx** [comment]
Hi Kai, thanks! If you have issues with getting the bundle to work, go ahead and post on Slack, and we'll get it to work :)
